TEST_APPS/device/socket_app/cmd_socket.cpp

```diff
@@ -299,9 +314,6 @@ static void sigio_handler(SInfo *info) {
 void cmd_socket_init(void)
 {
     cmd_add("socket", cmd_socket, "socket", MAN_SOCKET);
-    cmd_alias_add("socket help", "socket -h");
-    cmd_alias_add("socket --help", "socket -h");
```

Comment:
maybe last one would be still usefull - does test uses it?

Reply:
That one seems to be an artifact from porting. It isn't used nor implemented.